src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getUserById } from './services/userService';
+
+type Props = {
+  onSubmit: (todos: Todo) => void;
```

Comment:
The `onSubmit` prop is defined in the `Props` type but is not used in the `App` component. Consider removing it if it's not needed.